docs/reference/run.md

```diff
@@ -154,13 +154,11 @@ The operator can identify a container in three ways:
 -   UUID short identifier ("f78375b1c487")
 -   Name ("evil_ptolemy")
 
-The UUID identifiers come from the Docker daemon, and if you do not
-assign a name to the container with `--name` then the daemon will also
-generate a random string name too. The name can become a handy way to
-add meaning to a container since you can use this name when defining
-[*links*](../userguide/dockerlinks.md) (or any
-other place you need to identify a container). This works for both
-background and foreground Docker containers.
+The UUID identifiers come from the Docker daemon, and if you do not assign a
+name to the container with `--name` then the daemon will also generate a random
+string name too. The `name` can become a handy way to add meaning to a container
+since you can use this `name` when referencing the container within a Docker
```

Comment:
this does **not** work for random generated names; only for explicitly named (`--name`) containers, and *not* on the default bridge network